### PR TITLE
fix issue preventing login

### PR DESCRIPTION
**Pull request by kyleboe**

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class ApplicationController < ActionController::Base
   include Pundit
-  after_action :verify_authorized
+  # after_action :verify_authorized
```
Comment (nvick) [inline app/controllers/application_controller.rb]:
```suggestion

```

Comment (benjaminwood) [inline app/controllers/application_controller.rb]: Let's just remove this line completely

Comment (nvick) [reply app/controllers/application_controller.rb]: Let's just remove it

**Comment by kyleboe** [comment]
Removed it.

**Pull request merged**